pkg/configuration/configuration.go

```diff
@@ -0,0 +1,141 @@
+// Package configuration is in charge of the validation and extraction of all
+// the configuration details from a configuration file or environment variables.
+package configuration
+
+import (
+	"strings"
+	"time"
+
+	errs "github.com/pkg/errors"
+	"github.com/spf13/viper"
```

Comment:
I also needed to fetch this manually using `GO111MODULE=off go get github.com/spf13/viper`. It *is* in go.mod though, so I don't know why this is not working. It worked from the beginning in the terminal as well. This is really a mess when the terminal behaves differently from the vscode language server.